sites/public/pages/eligibility/disability.tsx

```diff
@@ -44,7 +44,7 @@ const EligibilityDisability = () => {
   const disabilityValues = [
     {
       id: "disabilityNo",
-      value: "no",
```

Comment:
Why not just use "true" and "false" for these values?

Reply:
The field values property expects strings and not booleans, so we would need to have a method here that converts between the two. I figured it would be cleaner for the context to directly store the value returned by the front-end widget and handle conversion at submission time. 

Additionally, if we use a boolean here, we would need to assign the "preferNotToSay" option to be disability=true to get the desired functionality, which makes it seem like we're assuming the user has a disability if they choose not to disclose.

Reply:
No, I was suggesting having it be the string value "true" or "false" for these, which alleviates the need to extract the string "no" into a shared string that you can then compare to when encoding it to the filter state. This is how the senior housing filter works in the modal.

Reply:
I updated to remove the shared string, though we're still checking string equality this time to "true". When we don't have the shared string variable, if someone updates how the disability values are stored in the future, this logic will break / they might not know they also need to update this equality check.

Reply:
What if we change the question options:
We could have options that are "no" and "yes, or prefer not to say" so they line up with what we actually need to know.

Reply:
And if someone only updates how this disability values are stored in one place, it'll break, right? So they'll notice somethings wrong?

Reply:
So you want to change the frontend to have the following options?:
- no
- yes, or prefer not to say

It feels like we are leaking implementation details to the user (they don't need to know we're treating the two options in the same way). It also equates the answer "yes" to "prefer not to say", which I don't think is good. If someone doesn't want to reveal their disability status checking "yes or prefer not to say" wouldn't respect that. 

The way I understand it, Will wanted me to remove the shared variable, but the point I'm trying to make is that a shared variable would still be useful here even when we change the strings to "true" and "false". If someone updates the disability values here, the filter url  in income.tsx will no longer be built correctly because we are checking a string literal there now.  The check is `=== "true"` rather than using a shared variable that if updated in one place will be updated for all files that use it.

Reply:
Yeah that's fair. My thought had been that having two options that do the same thing could be confusing as well, so whatever seems less confusing sounds fine.

I think I agree with you here re a shared variable. I'm not as worried about someone changing it and causing an error cause I think it'd be caught in testing, but I do think we should make clear in code that we're treating "yes" and "prefer not to say' the same.

Rather than having a single shared variable for "no" and a string for "yes", which is confusing to read at first, let's add an enum for the values like
```
enum disabilityOptions = {
  yes = "yes"
  no = "no"
  preferNotToSay = "preferNotToSay"
}
```
with the enum used like `value: disabilityOptions.no`, and then that gets used here and in `income.tsx`. That way all the values are together.

Then in income.tsx, we can add a comment explaining why we're only checking for no.

This way
- it's clear what the options for the disability question are
- they're all in one place
- if the way we handle the question in the future changes, we already have all the options wired up where they need to be

Reply:
That seems fine as a future refactor, you can file that one; we have true/false strings elsewhere as well but having it correspond to the actual boolean that ends up getting set will hopefully make it easier for future maintainers.

Reply:
Okay I'll create a bug for this.